go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/go-chi/chi/v5
 
-go 1.14
+go 1.16
```

Comment:
This downgrades chi's compatibility with older Go versions.

Can you please raise an issue first, so we can discuss pros/cons of dropping support for Go 1.14-1.15 and gather feedback from the community?

I don't disagree btw. It might be the right time to start requiring Go 1.16+.

Thanks!

Reply:
sure! 👍🏻

Reply:
done here: https://github.com/go-chi/chi/issues/963